backoffice/pages/catalog/brands/index.tsx

```diff
@@ -1,13 +1,36 @@
 import type { NextPage } from 'next'
 import Link from 'next/link'
 import React, { useEffect, useState } from 'react';
-import { Button, Table } from 'react-bootstrap';
+import { Button, Modal, Table } from 'react-bootstrap';
 import type { Brand } from '../../../modules/catalog/models/Brand'
-import { getBrands } from '../../../modules/catalog/services/BrandService'
+import { deleteBrand, getBrands } from '../../../modules/catalog/services/BrandService'
 
 const BrandList: NextPage = () => {
+    const [brandIdWantToDelete, setBrandIdWantToDelete] = useState<number>(-1);
+    const [brandNameWantToDelete, setBrandNameWantToDelete] = useState<string>("");
+    const [showModalDelete, setShowModalDelete] = useState<boolean>(false);
     const [brands, setBrands] = useState<Brand[]>([]);
     const [isLoading, setLoading] = useState(false);
+    const handleClose: any = () => setShowModalDelete(false);
+    const handleDelete: any = () => {
+      if (brandIdWantToDelete == -1) {
+        return;
+      }
+      deleteBrand(brandIdWantToDelete).then((response) => {
```

Comment:
Could you please help to integrate with toast as in category

Reply:
I resolve it